pyfr/integrators/dual/base.py

```diff
@@ -29,3 +121,97 @@ def _dual_time_source(self):
     @abstractmethod
     def finalise_step(self, currsoln):
         pass
+
+    @memoize
+    def prolrest(self, l1, l2):
+        prolrestkerns = proxylist([])
+
+        for etype in self.system.ele_types:
```

Comment:
Probably looping over the wrong thing, want to loop over l1eles, l2eles directly.